digdag-client/src/main/java/io/digdag/client/DigdagClient.java

```diff
@@ -569,14 +583,64 @@ public InputStream getLogFile(long attemptId, RestLogFileHandle handle)
 
     public InputStream getLogFile(long attemptId, String fileName)
     {
-        Response res = target("/api/logs/{id}/files/{fileName}")
-            .resolveTemplate("id", attemptId)
-            .resolveTemplate("fileName", fileName)
-            .request()
-            .headers(headers.get())
-            .get();
-        // TODO check status code
-        return res.readEntity(InputStream.class);
+        Invocation request = target("/api/logs/{id}/files/{fileName}")
+                .resolveTemplate("id", attemptId)
+                .resolveTemplate("fileName", fileName)
+                .request()
+                .headers(this.headers.get())
+                .buildGet();
+
+        return invokeWithRetry(request)
+                .readEntity(InputStream.class);
+    }
+
+    private Response invokeWithRetry(Invocation request)
+    {
+        Retryer<Response> retryer = RetryerBuilder.<Response>newBuilder()
+                .retryIfException(not(DigdagClient::isDeterministicError))
+                .withWaitStrategy(exponentialWait())
```

Comment:
This library looks cool. Do you think we'd better replace `io.digdag.util.RetryExecutor` with `com.github.rholder.retry.Retryer` in the future?